[2025-10-29T16:38:49.863+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_scd2.run_processor_harmonizer manual__2025-10-29T16:34:38.589633+00:00 [queued]>
[2025-10-29T16:38:49.902+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_scd2.run_processor_harmonizer manual__2025-10-29T16:34:38.589633+00:00 [queued]>
[2025-10-29T16:38:49.904+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-10-29T16:38:49.981+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_processor_harmonizer> on 2025-10-29 16:34:38.589633+00:00
[2025-10-29T16:38:50.008+0000] {standard_task_runner.py:60} INFO - Started process 1039 to run task
[2025-10-29T16:38:50.064+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_scd2', 'run_processor_harmonizer', 'manual__2025-10-29T16:34:38.589633+00:00', '--job-id', '262', '--raw', '--subdir', 'DAGS_FOLDER/process_scripts.py', '--cfg-path', '/tmp/tmpr68gdfaw']
[2025-10-29T16:38:50.096+0000] {standard_task_runner.py:88} INFO - Job 262: Subtask run_processor_harmonizer
[2025-10-29T16:38:50.253+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-29T16:38:50.493+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_scd2.run_processor_harmonizer manual__2025-10-29T16:34:38.589633+00:00 [running]> on host b5b5035930e9
[2025-10-29T16:38:50.683+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline_scd2' AIRFLOW_CTX_TASK_ID='run_processor_harmonizer' AIRFLOW_CTX_EXECUTION_DATE='2025-10-29T16:34:38.589633+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-29T16:34:38.589633+00:00'
[2025-10-29T16:38:50.689+0000] {logging_mixin.py:188} INFO - Exécution du script : /opt/***/scripts/processor_harmonizer.py
[2025-10-29T16:46:31.058+0000] {logging_mixin.py:188} INFO - 1227/5160 lignes mises à jour, 2977 lignes ignorées dans Extract_Station_20251029.xlsx (logs → data/out/Harmonize log/20251029T163907Z)
1100/90169 lignes mises à jour, 60822 lignes ignorées dans Invariants_details_20251029.xlsx (logs → data/out/Harmonize log/20251029T164205Z)
1100/90169 lignes mises à jour, 60822 lignes ignorées dans Invariants_study_20251029.xlsx (logs → data/out/Harmonize log/20251029T164557Z)
[2025-10-29T16:46:31.132+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-10-29T16:46:31.545+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_scd2, task_id=run_processor_harmonizer, execution_date=20251029T163438, start_date=20251029T163849, end_date=20251029T164631
[2025-10-29T16:46:32.180+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-29T16:46:32.886+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
